src/sentry/static/sentry/app/views/dataExport/dataDownload.tsx

```diff
@@ -0,0 +1,171 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {Client} from 'app/api';
+import {Config, Organization} from 'app/types';
+import {RouteComponentProps} from 'react-router/lib/Router';
+import AsyncView from 'app/views/asyncView';
+import withApi from 'app/utils/withApi';
+import withOrganization from 'app/utils/withOrganization';
+import withConfig from 'app/utils/withConfig';
+import withLatestContext from 'app/utils/withLatestContext';
+import {PageContent} from 'app/styles/organization';
+import {t} from 'app/locale';
+
+import Button from 'app/components/button';
+import Sidebar from 'app/components/sidebar';
+
+type RouteParams = {
+  orgId: string;
+  dataId: string;
+};
+
+type Payload = {
+  storage_url?: string;
+  expired_at?: string;
+  status?: 'EXPIRED' | 'INVALID' | 'VALID' | 'EARLY';
```

Comment:
You could use an [enum here](https://www.typescriptlang.org/docs/handbook/enums.html#string-enums)

Reply:
@markstory What's the advantage of using enum here? I haven't been using it for cases like this.

Reply:
If you need a switch/case statement, or comparisons in conditions you can re-use the enum instead of strings.

Reply:
Addressing this in my latest commits, thanks yall